SECURITY.md

```diff
@@ -0,0 +1,19 @@
+## Coordinated Vulnerability Disclosure Policy
+We ask security researchers to keep vulnerabilities and communications around vulnerability submissions private and confidential until a patch is developed to protect the people using Crypto.com’s protocols. In addition to this, we ask that you:
+
+- Allow us a reasonable amount of time to correct or address security vulnerabilities.
+- Avoid exploiting any vulnerabilities that you discover.
+- Demonstrate good faith by not disrupting or degrading Crypto.com’s data or services.
+
+## Vulnerability Disclosure Process
+Once we receive a vulnerability report, Crypto.com will take these steps to address it:
+
+1. Crypto.com will confirm receipt of the vulnerability report within 5 business days. The timing of our response may depend on when a report is submitted. As our daily operations are distributed in time zones across the globe, response times may vary. If you have not received a response to a vulnerability report from us within 5 business days, we encourage you to follow up with us again for a response.
+2. Crypto.com will investigate and validate the security issue submitted to us as quickly as we can, usually within 10 business days of receipt. Submitting a thorough report with clear steps to recreate the vulnerability and/or a proof-of-concept will move the process along in a timely manner.
+3. Crypto.com will acknowledge the bug, and make the necessary code changes to patch it. Some issues may require more time than others to patch, but we will strive to patch each vulnerability as quickly as our resources and development process allow.
+4. Crypto.com will publicly release the security patch for the vulnerability, and acknowledge the security fix in the release notes once the issue has been resolved. Public release notes can reference to the person or people who reported the vulnerability, unless they wish to stay anonymous.
+
+## Contact Us
+If you find a security issue, you can report it on the [Crypto.com HackerOne Bug Bounty Program](https://hackerone.com/crypto) or you can contact our team directly at [oss@crypto.com](mailto:oss@crypto.com).
```

Comment:
the bounty doesn't cover cronos play

Reply:
It is the same as https://github.com/cronos-labs/play-unreal-plugin/blob/main/SECURITY.md?plain=1#L17.

Should we remove it in play-unreal-plugin as well?

Reply:
@ricky321q

Reply:
Yeah better remove it if it not covered. @damoncro

Reply:
Removed.